Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -292,6 +293,12 @@ static String mutationObserverNotificationString()
     RefPtr { m_contentsLayer }->setNeedsDisplayInRect(contentsRect);
 }
 
+void UnifiedPDFPlugin::setNeedsRepaintInDocumentRects(OptionSet<RepaintRequirement> repaintRequirements, const Vector<FloatRect>& rectsInDocumentCoordinates)
+{
+    for (const auto& rectInDocumentCoordinates : rectsInDocumentCoordinates)
```

Comment:
`const ` is unnecessary and usually omitted in such cases.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/9355bda2fca30c7dd4b198ad7569cb367250efcb